### DISCUSSION TITLE
Type-graphql maintanance

### DISCUSSION THREAD

**Discussion by kkhanhluu** [MichalLytek/type-graphql#1052] [Q&A] [answered]
Thank you for your great project typegraphql. However, the last release is last year and i only see 3 commits in august and no commits since then. Is typegraphql still being maintained and when do you plan to release version 2?

**Comment by MichalLytek** [comment, marked as answer]
There are plenty of not released yet features:
https://github.com/MichalLytek/type-graphql/blob/master/CHANGELOG.md#unreleased

I see a lot of people still uses v0.17.6, so I try to publish bigger updates less frequently, in order to not force them to bump deps every week. `v1.2` needs some polishing and some other pending fixes and PRs before releasing, however I think I can publish prelease package on npm.

I'm in a process of moving into a new house so temporarily there might be less commit activity because I only have time to answer issues, discussion and stackoverflow. Most of my engagement is now focused on TypeGraphQL Prisma integration.

**Reply by kkhanhluu** [reply]
thanks for your reply
